core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -69,7 +69,8 @@ object Runtime {
           urlBase + cloudContext.asString + "/" + runtimeName.asString + "/" + tool.proxySegment
         )
       case _: CloudContext.Azure =>
-        throw new NotImplementedError("Proxying Azure runtime is not supported yet")
+        //TODO: this is not the correct proxy URL
+        new URL(urlBase)
```

Comment:
I don't think its a good idea to simply throw an exception here because it is used in two places that are invoked prior to proxy URL being implemented where runtimes are created (in DB and test code), i.e. this exception is hit whenever an Azure Runtime is created.

https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala#L790
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala#L267

Given the use case of this field is via leo proxy and consumers, I see little risk in including a dummy value with a future TODO.